Nos/Extensions/Font.swift

```diff
@@ -9,21 +9,37 @@ import SwiftUI
 
 // https://stackoverflow.com/a/74416073
 extension Font {
-    static var clarity = Font
-        .custom("ClarityCity-Regular", size: UIFont.preferredFont(
-            forTextStyle: .body
-        ).pointSize)
-    
-    static var clarityMedium = Font
-        .custom("ClarityCity-Medium", size: UIFont.preferredFont(
-            forTextStyle: .body
-        ).pointSize)
-    
-    static var clarityBold = Font
-        .custom("ClarityCity-Bold", size: UIFont.preferredFont(
-            forTextStyle: .body
-        ).pointSize)
-    
+    static var clarity = clarityRegular(.body)
+
+    static func clarity(_ fontWeight: UIFont.Weight, textStyle: UIFont.TextStyle = .body) -> Font {
```

Comment:
I created this function because it makes it easier to match the UI with the design on Figma